crates/polars-plan/src/logical_plan/lit.rs

```diff
@@ -456,7 +456,7 @@ impl Hash for LiteralValue {
                 s.null_count().hash(state);
                 const RANDOM: u64 = 0x2c194fa5df32a367;
                 let mut rng = (len as u64) ^ RANDOM;
-                for _ in 0..5 {
+                for _ in 0..std::cmp::min(5, len) {
```

Comment:
This adds back the `cmp::min`. But I can also do `if len == 0 { return }`, difference is just the number of hash steps for series with length `0 < n < 5`.